### ISSUE TITLE
What should be the focus of the Governance Working Group?

### ISSUE THREAD

**Issue by lastmjs** [dfinity/wg-governance#1]
During our first meeting we are going to discuss the focus and scope of the Governance Working Group. If you have an idea of what you believe should be the focus and scope of this working group, then please post it as a comment on this issue. We'll use reactions to come to consensus on what the working group believes should its focus and scope.

**Comment by paulyoung** [comment]
One idea would be to put some focus on motion proposals that have passed but not been implemented.

**Comment by Rehbaum** [comment]
Ethics for me which means centralization but then we need governance and then voting will create decentralization.

We have to fix the start of each problems not band-aid the result:  
**Whale**
The difference between a whale and everyone else: **NONE**

A person or group that invested a large amount of money and has many more tokens than most. The whole purpose of Dfinity is to raise capital and the whale and small investors achieved this. The whale has taken on more risk than most and would lose just like a small investment.

**Inactive**
A group of people invested in ICP that have the right to be inactive.

**Dead**
People who may have passed and the family will make decisions on the future of the neuron.

These are Ethics and no decentralization applies.

**dwarf**
Those who oppose, blunder, disrupt and  provide disinformation.

**Comment by dprats** [comment]
**_(Meta context: I have shared the feelings below with others in the community. Despite my role in the DFINITY foundation, i want to highlight, I am NOT the DFINITY representative to this WG, so these are my thoughts, albeit strongly opinionated)_**

# TLDR:

"Governance" has many meanings and usages in blockchain and IC space. I will lay out the landscape of uses I see and why I believe **strongly** this group should start with a well-defined scope of it. I think this WG should focus on narrowly on "NNS mechanisms" and leave other issues (e.g tokenomics) to other WGs.

## 1. Landscape of uses

“Governance” has multiple meanings in IC ecosystem:

### a. On-chain changes:
*  Tokenomics changes (e.g. minting, burning)
* Changes to NNS mechanisms to improve participation (eg. voting rewards, spam, neurons, following, etc..)
* Subnet management (e.g. node providers)
* Changes to the IC protocol code

### b. Off-chain community consensus-building:
* Agreeing on roadmap of the IC
* Agreeing on standards 

## 2. Why I believe the group should narrowly focus on "Changes to NNS mechanisms to maximize NNS governance participation"

I believe a well-defined scope will be helpful for the success of this working group as well as FUTURE standards. My proposal is that THIS Governance Working Group focus on NNS mechanisms for participation (voting, rewards, neurons, etc...) and see itself (in a flawed analogy) as "the Constitutional convention and NOT the congress." I would like this group to focus on questions like:

* Should NNS periodically expire following relationships?
* Should there be a new NNS topic?
* Should reward weights change?
* etc...

But most importantly, I would like this group to do the following:

a. Help define a holistic vision or goal of what the _ideal NNS mechanisms_ should look like
b. Prioritize issues and systematically work through them
c. Analyze and iterate based on what the WG observes to see how previous actions have performed 

# 3. What I think this working group should NOT focus on

To help clarify, I suggest the WG NOT work on the following:

a. Tokenomics - I think this should be a different WG, so questions like "what is the ideal minting/burning rate?" should be part of a tokenomics WG and not part of THIS WG.

b. Standards - I do not think this WG should discuss specific standards, but those should be the discussions of specific WGs.

c. Roadmap - Perhaps more controversially, I believe this WG should focus on HOW to make community decisions, not on which decisions these should be. i think Roadmap features should be other WGs where domain specialists and interested entities can participate... I believe if this WG's scope goes into "node infrastructure roadmap" or "II roadmap" or "consensus protocol roadmap" it can wildly escape scope.

# 4.  A potential vision of the NNS voting system

I think this WG should create a concrete vision of the NNS voting system and help get us there. To kickstart this conversation, I think an ideal NNS achieves these goals:

**a. Decentralized & active:** The governance is decentralized. That is, the voting power is distributed over many, independent entities and there is not one single or a few entities that can decide by themselves how the Internet Computer evolves. Also, there is a broad and active community meaning that many participants regularly interact with the governance and there is a variety of active voters (known neurons).

**Secure & available:** Malicious proposals, that would be against the interest of the majority, cannot bypass the voting process and be “voted through” by exploring limitations of the voting protocol. Moreover, the governance system is always available to take new proposals and process them.

**Long-term thinking:** Voters are incentivized to vote in the long term interest of the Internet Computer.

**Efficient & scalable:** The voting process remains practicable when the Internet Computer and its community increase in size over time. This includes ease of deployment (change is easy to program and test). Also, the voting process can be completed efficiently by voters. For example, it is efficient to identify worthy proposals and easy to vote.

Reactive: High priority proposals, for example urgent security patches, can be adopted and executed quickly.

**Purposeful:** Proposals should not be misused/bogus. 
* For example not Skew rewards without meaningful proposals
* Bad content, display bad/inappropriate content to users

**Simple & accessible:** There is a low barrier for entry, meaning that users with limited commitment (small staked amounts, low dissolve delay) can propose new ideas and contribute to the discussion. The voters understand how the governance system works.

**Comment by wpb1** [comment]
Here are a few ideas that can be considered for objectives of this working group...

1) Develop a vision about how the NNS will achieve decentralization.  This includes short, medium, and long term goals.  Who is expected to be contributors to the IC and how do they become contributors?  What does it mean to be a contributor?

2) Shape an explanation of the purpose of tokenomics.  What are we trying to achieve through tokenomics and why is it important to the NNS?

3) Define a path to implementation for community driven proposals.  How should DFINITY be involved or not involved?  Is the community expected to implement their own proposals?  How can the community implement proposals? 

4) If experts are required to make smart governance decisions, then define who is an expert.  What are the qualifications?  What expert roles are needed and why would an expert get involved?  

Also, if experts are going to be on center stage in long term NNS governance decisions, then should any protections exist from the peanut gallery. Top experts could easily be disinterested in the spotlight, unwilling to engage in public conversation, and/or be intimidated by criticism.  Will it require a support network to attract and retain them or is everyone on their own?

**Comment by brutoshi** [comment]
> **_(Meta context: I have shared the feelings below with others in the community. Despite my role in the DFINITY foundation, I want to highlight, I am NOT the DFINITY representative to this WG, so these are my thoughts as a member of the IC community, albeit strongly opinionated)_**
> 
> # TLDR:
> "Governance" has many meanings and usages in blockchain and IC space. I will lay out the landscape of uses I see and why I believe **strongly** this group should start with a well-defined scope of it. I think this WG should focus on narrowly on "NNS mechanisms" and leave other issues (e.g tokenomics) to other WGs.
> 
> ## 1. Landscape of uses of word “governance”
> In my experience “Governance” has multiple meanings in blockchain and IC ecosystem, and this is further compounded that the IC is a blockchain with on-chain self-updating (NNS can vote to change its own code). Here is what i have seen:
> 
> ### a. On-chain changes:
> * Tokenomics changes (e.g. minting, burning)
> * Changes to NNS mechanisms to improve participation (eg. voting rewards, spam, neurons, following, etc..)
> * Subnet management (e.g. node providers)
> * Changes to the IC protocol code
> 
> ### b. Off-chain community consensus-building:
> * Agreeing on roadmap of the IC
> * Agreeing on standards
> 
> ## 2. Why I believe the group should narrowly focus on "Changes to NNS mechanisms to maximize NNS governance participation"
> I believe a well-defined scope will be helpful for the success of this working group as well as FUTURE working groups. My proposal is that THIS Governance Working Group focus on NNS mechanisms for participation (voting, rewards, neurons, etc...) and see itself (in a flawed analogy) as "the Constitutional convention and NOT the congress." I would like this group to focus on questions like:
> 
> * Should NNS periodically expire following relationships?
> * Should there be a new NNS topic?
> * Should reward weights change?
> * etc...
> 
> ### But most importantly, I would like this group to do the following:
> a. Help define a holistic vision or goal of what the _ideal NNS mechanisms_ should look like b. Observe & measure the interaction between NNS and community c. Analyze and decide the gap between "the ideal version of NNS + community looks like" and reality. b. Prioritize issues and systematically work through them to bridge gap between vision and reality
> 
> # 3. What I think this working group should NOT focus on
> To help clarify, I suggest the WG NOT work on the following:
> 
> a. Tokenomics - I think this should be a different WG, so questions like "what is the ideal minting/burning rate?" should be part of a tokenomics WG and not part of THIS WG.
> 
> b. Standards - I do not think this WG should discuss specific standards, but those should be the discussions of specific WGs.
> 
> c. Roadmap - Perhaps more controversially, I believe this WG should focus on HOW to make community decisions, not on which decisions these should be. i think Roadmap features should be other WGs where domain specialists and interested entities can participate... I believe if this WG's scope goes into "node infrastructure roadmap" or "II roadmap" or "consensus protocol roadmap" it can wildly escape scope.
> 
> # 4. A potential vision of the NNS voting system
> I think this WG should create a concrete vision of the NNS voting system and help get us there. To kickstart this conversation, I think an ideal NNS achieves these goals:
> 
> **a. Decentralized & active:** The governance is decentralized. That is, the voting power is distributed over many, independent entities and there is not one single or a few entities that can decide by themselves how the Internet Computer evolves. Also, there is a broad and active community meaning that many participants regularly interact with the governance and there is a variety of active voters (known neurons).
> 
> **Secure & available:** Malicious proposals, that would be against the interest of the majority, cannot bypass the voting process and be “voted through” by exploring limitations of the voting protocol. Moreover, the governance system is always available to take new proposals and process them.
> 
> **Long-term thinking:** Voters are incentivized to vote in the long term interest of the Internet Computer.
> 
> **Efficient & scalable:** The voting process remains practicable when the Internet Computer and its community increase in size over time. This includes ease of deployment (change is easy to program and test). Also, the voting process can be completed efficiently by voters. For example, it is efficient to identify worthy proposals and easy to vote.
> 
> **Reactive**: High priority proposals, for example urgent security patches, can be adopted and executed quickly.
> 
> **Purposeful:** Proposals should not be misused/bogus.
> 
> * For example not Skew rewards without meaningful proposals
> * Bad content, display bad/inappropriate content to users
> 
> **Simple & accessible:** There is a low barrier for entry, meaning that users with limited commitment (small staked amounts, low dissolve delay) can propose new ideas and contribute to the discussion. The voters understand how the governance system works.

I talked to some serious ICP stakeholders in the Asia community (full-time builders and ICP investors “8 Year Gang”) regarding this WG. 

There are some good suggestions and questions on NNS improvement, such as making a dedicated website to record NNS updates for normal users (from D+ Community’s Mike https://twitter.com/D_PlusCommunity), the handling of silent neurons, etc. These issues should definitely be discussed in WG. I have invited them to join the WG meeting. 

However, there is also a common concern regarding Dfinity’s Roadmap and R&D priority. I won’t go into specific details which have been raised in the forum or on other social media. I think the WG can also work on how to effectively collect these concerns (with some structure going through NNS), acting like “election polls”, so the Dfinity Leadership can listen.

**Comment by bjoernek** [comment]
I would suggest that we spend some time in the working group on discussing a set of concrete proposals and how to refine them. For example

1. **Manual voting** (motion proposal https://dashboard.internetcomputer.org/proposal/38985): The overall intent and benefit of the proposal is clear. However, it is not yet clear (at least to me), how it would be implemented. For example how would this interact with wait-for-quiet. 
2. **Periodic confirmation of followees** (motion proposal https://dashboard.internetcomputer.org/proposal/55651): What is the way forward on this proposal? (e.g, with respect to allowing for a reactive system, as discussed prior in the forum).

**Comment by KrackersICP** [comment]
Hi Everyone, 

Congrats Jordan, you made me get a Github account!  I'm not a developer but believe the fate of ICP is directly correlated to getting the governance right...governance is what I'm most passionate about and what attracted me to the ICP protocol in the first place (The NNS Specifically). 

**Why I Care** - I believe that if humans do not collectively evolve the current governance systems, born of the enlightenment, what will follow is a period of sustained decline for humanity (think dark ages 500-1500 AD).  Our world has changed so much in such a short time with the rise of the digital/exponential age.  Governance has primarily not changed and this is leading to governance problems globally.  

I see ICP as a potential tool and model for 21st century governance.  So hope I'm able to make a modest contribution to the effort and appreciate the opportunity.

Agree with @lastmjs and @dprats that the right analogy for this working group is a Continental Congress.

For non-US participants, it may be worth providing a few bullet points on this process which led to the governance system of the United States

- There were two Continental Congresses in formation of the USA. 
- Each one's goal was to design and implement a federal governance system for what is now the USA. 
- First one produced the Articles of Confederation in 1777 which was first try at USA self governance but had structural problems which led to government not working (maybe where ICP is at right now)
- The 2nd one was in 1787 to fix the problems with the Articles of Confederation.  They designed a new constitution which is still the governance framework of the USA to this day.  

This groups focus and scope should be on the governance framework/design for the ICP.  I would propose producing a ICP Constitution that is eventually codified into the NNS should be our loftiest goal.  

I see this 'Constitution' having the following elements 
1.  Define the nature and boundaries of the IC community - what qualifies someone/something to be considered apart of ICP?
2.  Declare the authority of the IC community - What can/can't ICP do as a governance body. 
3.  The structures and processes of IC governance - How we vote, change things, etc. 
4.  The rights & responsibilities of the users of the IC - What rights, if any do community members have?  What is minimum participation level 
5.  Establishes the political institutions of IC - NNS, Legal, Social, Defense, Executive, etc. 
6.  Distributes power between those political institutions - Checks and balances within the system

 Look forward to participating on the call tomorrow.

**Comment by dprats** [comment]
At the first session of the Governance working Group, @brutoshi echoed what he said above. He had a great point. After a brief discussion, we thought a good way to incorporate the feedback is to add an 8th property of the ideal NNS. For completion and context, I have added it to he bottom of this properties below:

> think this WG should create a concrete vision of the NNS voting system and help get us there. To kickstart this conversation, I think an ideal NNS achieves these goals:
> 
> **1. Decentralized & active:** The governance is decentralized. That is, the voting power is distributed over many, independent entities and there is not one single or a few entities that can decide by themselves how the Internet Computer evolves. Also, there is a broad and active community meaning that many participants regularly interact with the governance and there is a variety of active voters (known neurons).
> 
> **2. Secure & available:** Malicious proposals, that would be against the interest of the majority, cannot bypass the voting process and be “voted through” by exploring limitations of the voting protocol. Moreover, the governance system is always available to take new proposals and process them.
> 
> **3. Long-term thinking:** Voters are incentivized to vote in the long term interest of the Internet Computer.
> 
> **4. Efficient & scalable:** The voting process remains practicable when the Internet Computer and its community increase in size over time. This includes ease of deployment (change is easy to program and test). Also, the voting process can be completed efficiently by voters. For example, it is efficient to identify worthy proposals and easy to vote.
> 
> **5. Reactive**: High priority proposals, for example urgent security patches, can be adopted and executed quickly.
> 
> **6. Purposeful:** Proposals should not be misused/bogus.
> 
> * For example not Skew rewards without meaningful proposals
> * Bad content, display bad/inappropriate content to users
> 
> **7. Simple & accessible:** There is a low barrier for entry, meaning that users with limited commitment (small staked amounts, low dissolve delay) can propose new ideas and contribute to the discussion. The voters understand how the governance system works.

**8. Representative of the IC Community's interest** - an ideal NNS system broadly represents the community's interests. This is a subjective notion of course (even defining who is part of the community is fuzzy), but the WG can serve as a form of watchdog which can take the same stance it does for the other properties: it can compile data to periodically grade how far a property is from its ideal... and then recommend solutions.

**Comment by legendface66** [comment]
tl;dr:
Let's start with a mission statement, then survey NNS participants and community members about issues they see with governance (this group is a good place to start), then define our scope by relating our mission statement to the issues we have identified.

Anyone who wants to jump on a call to discuss the contents of this post, please DM me. We can create a presentation for the group to consider at the next meeting.

******

I think it's better to define our scope against real-world issues. [This](
https://docs.google.com/spreadsheets/d/1UmMiJra5skQCoUn98TDbdmmyTTj4yK55WV4b9F6_4UU/edit?usp=sharing) is a link to the results of a survey I did last year (trigger warning, it's pretty aggro). I think we should do a new one but this is here as an example. The issues raised at the time were:

Seed rounders report not being technical enough to access their neuron and vote.
Preconfiguring neurons to follow DFINITY meant they have total control.
Poor investor confidence and poor launch led to seed rounders not extending their lockup times.
Long lockup times dissuade investors from locking tokens.
Uncertainty of future plans dissuades locking up.
If the described issues were resolved investors would lock up for longer
Given the rewards, locking for 8 years is the only thing that makes sense for some investors
If there were rewards for locking up for less than 6 months investors would take advantage of this
Some investors view 6 months as way too long.

If we started with a mission statement, for example:

**To encourage investor participation in the IC governance system and ensure community representation.**

Then we could derive from the issues identified that our initial scope would be:

- Advising on the seed round NNS interface
- Advising on default neuron follows
- Advising on investor relations
- Advising on roadmap publication
- Advising on rewards and lock up times

I believe we should begin with evidence before we talk. All our activities should be grounded in solving visible problems and have clear success metrics. Otherwise we will be spinning our wheels.

As for our process, I don't think we should discuss as a large group. Instead, a small group should get together and define a process based on the myriad of examples of multistakeholder governance already out there. We want to adopt a process as soon as possible so that work can begin. Anything is better than nothing and given the size of the group, responding to all opinions will slow this down. Not to imply the group's opinions are not valid, just that a small group should be able to come up with something good enough and speed is the priority. We can fix issues as we encounter them.

Finally, by surveying this group one person can identify all the issues and collate them rather than creating long discussion threads that we all need to read through.

**Comment by bjoernek** [comment]
Hi @dprats, 

I have some comments on the additional goal which you proposed

> 8. Representative of the IC Community's interest - an ideal NNS system broadly represents the community's interests. This is a subjective notion of course (even defining who is part of the community is fuzzy), but the WG can serve as a form of watchdog which can take the same stance it does for the other properties: it can compile data to periodically grade how far a property is from its ideal... and then recommend solutions.

What I like about this goal
- I agree that we should build & enhance the governance system in such a way that it reflects the interest of the community of the IC. 
- I also makes a lot of sense to me to collect as part of the working group current pain points of the governance system, as suggested by @brutoshi.

Why I believe that this meta goal does fit into the above list 
- The above list is a draft collection of concrete goals/properties which  a governance/voting system should have. Based on this list we can evaluate concrete suggestions in a structured way (e.g. by checking whether a proposal makes the system more secure and available). 
- The interest of the community should be expressed by these goals. Thus it is a  concretisation of the interest of the community. 
- Without making the goal "interest of the community" more concrete, it would be difficult to assess it. 

Concluding I suggest that the goal "Representative of the IC community's interest" should be a preamble to the list of goals, but not an element of the list.

**Comment by DataaDave** [comment]
Hey @legendface66 

> tl;dr: Let's start with a mission statement, then survey NNS participants and community members about issues they see with governance (this group is a good place to start), then define our scope by relating our mission statement to the issues we have identified.

I think this is a viable option to get input from the less extroverted people as well. Surveys can be done in between meetings so it wouldn't slow us down.

Regarding the already listed goals it is important that they are measurable.  E.g. desirable Nakamoto Coefficient regarding decentralization.

**Comment by Rehbaum** [comment]
**Governance**
Governance **encompasses the system** by which an organization is controlled and operates, and the mechanisms by which it, and its people, are held to account. 

@dprats I have seen your handling of this subject and many others and I have little faith. You can’t departmentalise this subject on your say so.

In any subject we will find conversations stray on all aspects in a given subject and trying to define a group of governance issues doesn’t sit right for me. 

Mission statements, roadmaps and most of the ideas here are already in place, many of these topics have been spoken about already. The only notable change for me is that the people with more self interest have lowered their enthusiasm and other personalities have come forward, over time. This is normally because these self interested have found a subject that does inspire them. 

Governance, Decentralization are great conversations for management and accountants but do not inspire much else. In my opinion the reason most people came on board this project was not the politics but the idea and therefore like many business ideas we need to extend the services that this project has and what it can deliver but we must understand the neuron owners and their interests.

**Let's talk about human nature**
I will use the analogy for driving a vehicle and the class type and abilities of each class which I believe we can all understand.

-----------------------------------------------------------------------------------------------------------------------------

**Entry Point**
Firstly we have the Learner class, they know that they want to drive a vehicle but don't have the competence, they will get the understanding through documentation but appling that behind the drivers wheel is a whole different experience.

Design
What the Learners don't understand is the technology that surrounds them that was designed by highly qualified people, some more than others, that protect and guide that individual.

Most neuron holders don't have the expertise and knowledge but we trust the infrastructure that is built around us.

------------------------------------------------------------------------------------------------------------------------------

**Provisional Driver or as I call then “No all, Know nothing”**
Then we have the provisional driver who now thinks they not only know how to drive but modify the vehicle but really their knowledge will only take away from the original design and then explains how clever they are.

--------------------------------------------------------------------------------------------------------------------------

**The Full Driver**
This driver has full abilities to drive but still very little experience

--------------------------------------------------------------------------------------------------------------------------

**The Advanced **
These are those who are highly qualified and move on to drive things that fly, float and need advanced knowledge.

-------------------------------------------------------------------------------------------------------------------------

The NNS was built by highly specialized people and let the **Entry Point and Know all, No nothing** make the decisions to modify the systems.

Just because a bike rider does not like gloves they get the experience and understanding when they fall over and use their hands to break the fall. 

Why should we allow changes to be made by these experts?

If you want new neuron HODL then maybe we should speak more clearly and say holders, provide incentive in aspects of the project that would attract new and old neuron owners to participate in something they know about.

I put forward the idea that Dfinity should have their own Internet engine for searches on their IC and that we should move the discussions on the internet instead of a closed forum that isn’t displayed from the NNS and we as a community could vote and develop for rewards in a public arena.

Hello Discord!!! We have been ripped off and could have been in discussions and rewards to build this engine??????????

How about we stop pandering to ETH and BTC and find a couple of other coins we can chain or smart contract too where us as neuron holders can invest our maturity and extra ICP into and have insight into the coins.

**Comment by Matlor** [comment]
Hello everyone

I’m a bit late with this post but I hope it benefits the discussion. First of all a brief thanks to the person mentioning Continental Congress, I didn’t even know that this existed (I’m not from the US).

I think we could categorize the points brought up so far into the following categories.

**Decentralization**
Many ideas that were mentioned such as long-term orientation are ways to achieve some form of decentralization. 
As it is such a vague and overused term it seems crucial that we properly define what we mean by decentralization. Personally, I think it’s definitely not something binary but rather describes how heavily users (mainly developers right now) are at risk when they commit resources to the system. As such I think it’s very useful to both understand the “hold-up problem” and theory of "incomplete contracts". We should plan to have an extensive discussion on decentralization.

**Concrete improvements in the present**
Includes things like: manual voting, followee confirmation, reward weights, dedicated website, participation, etc.

**Preventing the system from being dysfunctional**
Includes things like efficient, scalable, secure, available, reactive, simple, accessible, etc.

**Decisions requiring expertise**
We are assuming here that certain decisions will create more value if certain people make them. We’re trading the benefit of their expertise for the risk of them acting in a selfish way. As such this seems to be something we should tackle once we have defined decentralization properly. 

**Rights, identity & institutions**
The problem that I see here is that the question of identity has not been addressed yet. In Blockchains our identities are just key pairs. I’m unsure if there can be rights/obligations and specialized institutions (e.g. defense etc.) without a more advanced notion of identity. Essentially our physical existence is not “at stake” in these systems yet. I believe this to be the core difference between countries and blockchains.

While we can definitely try to tackle the "Concrete improvements" and "Preventing the system from being dysfunctional" categories right now, it feels important that we first build an understanding of what we are working towards (starting with decentralization).  


In addition to these thoughts here are the properties I think an ideal NNS should have:

**a. One-person-one vote (while ensuring early backers get rewarded for their efforts)
b. Maintains the long-term orientation of voters (even with one-person-one)
c. Actively incentivizes (disincentivizes) positive (negative) externalities
d. Has moral values built into it (that are hard to get rid of even with votes)
e. Enables relational contracts**

a) Contains 2 unsolved problems. 1) Identity. 2) How can we distribute ownership to “everyone” without early creators/backers being screwed over.

b) I think Switzerland does this by requiring someone to stay within the same municipality for many years before they become eligible to become Swiss. All local assets that are unlikely to move out of the country (e.g. social capital) prevent people from voting short-term oriented (and then potentially leaving the country). I’m a friend of more inclusive systems though, maybe we can do better than Switzerland.

c) The NNS would be analogous to real governments instead of other Blockchains. Whatever benefits the ecosystem should be funded by the NNS. Whatever negatively affects the ecosystem should be disincentivized. 

d) I don’t think anything governed by people is neutral. We therefore won’t get around defining core values (e.g. human rights). I’m not yet sure how this could be built into such a system though.

e) This is something really underrated. Current legal systems are not fully built with relational contracts in mind. They allow parties to form contracts while only defining broad intentions around their long-term relationship. It would be in the hands of the NNS or sub-structures (judges) to help such parties to govern their relationships. As being the structure most resilient to bribes/attacks the NNS is best suited for that.

I hope these thoughts are not too broad. Looking forward to the coming discussion.

**Comment by dprats** [comment]
> @dprats I have seen your handling of this subject and many others and I have little faith. You can’t departmentalise this subject on your say so.

Hi there, I just posted a proposal (to be discussed within the working group) one ONE way to slice the world into something concrete, I did not intend anything to happen on my "say so." Indeed, that would be quite naive of when trying to build consensus in a decentralized protocol. I hope my intent was evident.

Regarding the rest of your post, I read some things that made sense ("we must understand the neuron owners and their interests") and others I do not quite follow (e.g. the connection between a search engine and Governance Working Group). 

I hope you attend the working group discussions as well to expand your vision. Group benefits from spirited visions.

**Comment by dprats** [comment]
> The problem that I see here is that the question of identity has not been addressed yet. In Blockchains our identities are just key pairs. I’m unsure if there can be rights/obligations and specialized institutions (e.g. defense etc.) without a more advanced notion of identity. Essentially our physical existence is not “at stake” in these systems yet. I believe this to be the core difference between countries and blockchains.

@Matlor not trying to derail your overall points on Governance (many of which I think I agree with), but your interest in identity made me want to ask: are you aware of the Identity Working Group?  They meet monthly and I think (if you haven't already) should attend and share thoughts on Identity there so they are not just lost on this thread: 

https://forum.dfinity.org/t/technical-working-group-identity-authentication/11902

**Comment by legendface66** [comment]
Document on smaller working groups:
https://docs.google.com/document/d/1c_Y3SPYgg2ceea46EcxavKHJII88UoO5Gq9kiDsYGQc/edit?usp=sharing

**Comment by legendface66** [comment]
[IC Governance WG: Issues & Mission Statement](https://forms.gle/rBBTmiWgZSgBreZA7)

[Short notes on Meeting #2](https://docs.google.com/document/d/1wgoHR_wmUT6pi3za2derOpEMe0xxUmfdwgpkpeiuz4s/edit?usp=sharing)

**Comment by legendface66** [comment]
Issues raised in the second Governance Working Group meeting were:

How do we prioritise roadmap items for DFINITY?
How do we initiate new working groups?
How do we coordinate with other groups?
Can the NNS implement multiple choice questions?
Should ICP Tokenomics be part of this group?
Can we discuss NNS mechanisms here?
Can we discuss what APIs are open access?

[Brief meeting notes](https://docs.google.com/document/d/1wgoHR_wmUT6pi3za2derOpEMe0xxUmfdwgpkpeiuz4s/edit?usp=sharing)

**Comment by Rehbaum** [comment]
To extend on my last suggestion. 

The first concern I have with any Governance working group is that we have a range of qualified persons to be included. 

That each group member can be voted on or off by other members “of that group” and that they give an explanation of why they are interested, would be committed, personal reasons and show their expertise.

As Governance is a broad subject then there should be different workgroups 

Examples
Network systems
Network management
Network security
Systems security
Secure hardware
Cryptography
Distributed systems
Economics

We all can’t be experts in all things.

Unless of course you are talking about a governance group to head the discussions of these subjects that can oversee the sub groups that include qualified persons for each subject.

With all neuron owners able to vote on proposals that each workgroup has defined without the noise of the less qualified.

**Comment by legendface66** [comment]
Hey @Rehbaum it would be helpful to have your input in the mission statement and addressable issues survey

[IC Governance WG: Issues & Mission Statement Survey](https://forms.gle/rBBTmiWgZSgBreZA7)

All input would be valuable @paulyoung @dprats @lastmjs @DataaDave @Matlor @bjoernek @KrackersICP @wpb1

**Comment by lastmjs** [comment]
> Document on smaller working groups: https://docs.google.com/document/d/1c_Y3SPYgg2ceea46EcxavKHJII88UoO5Gq9kiDsYGQc/edit?usp=sharing

I'm pretty sure we should do something like this :) We'll discuss it today. I think we should not use Docusign but pull requests.
